src/ostorlab/runtimes/definitions.py

```diff
@@ -298,6 +317,72 @@ def from_bus_message(cls, request):
         return cls(agent_settings, name, description)
 
 
+@dataclasses.dataclass
+class AssetsDefinition:
+    targets: List[base_asset.Asset]
+    name: Optional[str] = None
+    description: Optional[str] = None
+
+    @classmethod
+    def from_yaml(cls, group: io.FileIO):
+        """Construct TargetGroupDefinition from yaml file.
+
+        Args:
+            group : target group .yaml file.
+        """
+        target_group_def = loader.load_target_group_yaml(group)
+        assets = target_group_def["assets"]
+        android_store_assets = assets.get("androidStore", [])
+        android_aab_file_assets = assets.get("androidAabFile", [])
+        android_apk_file_assets = assets.get("androidApkFile", [])
+        ios_store_assets = assets.get("iosStore", [])
+        ios_file_assets = assets.get("iosFile", [])
+        ip_assets = assets.get("ip", [])
+        domain_assets = assets.get("domain", [])
+        link_assets = assets.get("link", [])
+        sbom_assets = assets.get("sbom", [])
```

Comment:
We don't have SBOM as an asset. Please stick with the assets as defined in the cli/scan/run/assets.